oak-run/src/main/java/org/apache/jackrabbit/oak/run/NamespaceRegistryCommand.java

```diff
@@ -122,19 +122,19 @@ protected Root getWriteRoot() {
             };
             if (analyse || fix) {
                 NamespaceRegistryModel registryModel = NamespaceRegistryModel.create(root);
-                if (fix) {
-                    Map<String, String> additionalMappings = new HashMap<>();
-                    if (mappings != null) {
-                        for (String mapping : mappings) {
-                            String[] parts = mapping.split("=");
-                            if (parts.length != 2) {
-                                System.err.println("Invalid mapping: " + mapping);
-                                return;
-                            }
-                            additionalMappings.put(parts[0].trim(), parts[1].trim());
+                Map<String, String> additionalMappings = new HashMap<>();
```

Comment:
Just checking: removing the check for "fix" is intended?

Reply:
It hasn't been removed, but moved. Yes, that is intended.